src/test/java/org/example/c03arrays/delete/DeleteElementAtPositionInArrayTest.java

```diff
@@ -14,7 +14,7 @@ public void givenPosition2ForDeleting3ShouldDisappeared() {
     }
 
     @Test
-    public void givenPosition1ToBeDeletedIn1ElementArrayShouldReturnAnEmptyArray() {
+    public void givenPosition1ToBeDeletedPositionIs0() {
```

Comment:
Wrong test name. given position to be deleted is 0

Reply:
Done